### DISCUSSION TITLE
Check that all undocumented procedures has internal access modifier

### DISCUSSION THREAD

**Discussion by vody** [StefanMaron/BusinessCentral.LinterCop#27] [New Rule]
This warning will push developers to make sure all public extension API is documented or to make sure that it is not public. It's even more important for AppSource apps as all public procedures are locked for future modifications if they are public so will act as an additional notification to set internal access modifier if it's not intended to be an app public API.

**Comment by someC0d3r** [comment]
Hey @vody 

Can you give me one or two AL coding examples in which the code is ok and not ok. :)

Greetings
Kev
